modules/op-release-notes-1-17.adoc

```diff
@@ -0,0 +1,168 @@
+      test2.default-revision: "stable"
+      test2.scm-type: "gitlab"
+      test2.server-url: "api.internal-gitlab.com"
+      test2.api-token-secret-name: "test2-secret"
+      test2.api-token-secret-key: "pat"
+      test2.api-token-secret-namespace: "test2"
+      test2.default-org: "tektoncd-infra"
+# ...
+----
++
+.Example of using a `test1` configuration in a pipeline run
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  name: example
+spec:
+  pipelineRef:
+  	resolver: git
+ 	  params:
+  	# ...
+  	- name: configKey
+   	  value: test1
+# ...
+----
+
+* With this update, you can now monitor the running `PipelineRun` resources at the namespace, pipeline, and pipeline run level, in addition to the cluster level. You can configure the level through the `TektonConfig` custom resource by setting the `metrics.running-pipelinerun.level` parameter value to `namespace`, `pipeline`, or `pipelinerun`. The default value is empty, which sets the monitoring at cluster level.
++
+.Example of monitoring the running pipeline run at the namespace level
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  pipeline:
+    # ...
+    metrics.running-pipelinerun.level: namespace
+# ...
+----
+
+[id="cli-new-features-1-17_{context}"]
+=== CLI
+
+* With this update, you can run the `skopeo-copy` command with additional arguments, for example, `--all` and `--preserve-digests`. You can pass these arguments as a space-separated string, as shown in the following example:
++
+.Example command
+[source,terminal]
+----
+$ tkn task start skopeo-copy ARGS=”--all --preserve-digests”
+----
+
+[id="pac-new-features-1-17_{context}"]
+=== {pac}
+
+* With this update, the `pipelines_as_code_running_pipelineruns_count` metric is added to show the number of running `PipelineRun` resources in {pac}.
+
+* With this update, the `pipelines_as_code_pipelinerun_duration_seconds_sum` metric is added to show the total duration in seconds taken by all `PipelineRun` resources in {pac}. The metric can be filtered by repository or by namespace.
+
+[id="tekton-chains-new-features-1-17_{context}"]
+=== {tekton-chains}
+
+* With this update, you can configure the `TektonConfig` custom resource (CR) to generate the `x509` key pair of `ecdsa` type and use it with {tekton-chains} to sign artifacts. You can generate the key pair by setting the the `generateSigningSecret` field in the `TektonConfig` custom resource (CR) to `true`:
++
+.Example of creating ecdsa key pair
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+# ...
+  chain:
+    disabled: false
+    generateSigningSecret: true 
+# ...
+----
+
+* With this update, the default values for `artifacts.taskrun.storage` and `artifacts.pipelinerun.storage` {tekton-chains} parameters are changed from `tekton` to `oci` to enhance the initial setup and performance of {tekton-chains}.
```

Comment:
The values are not changed, if user has no default values, then default values are set, and also we are setting default values for `artifacts.oci.storage`

cc @jkhelil can you please elaborate a bit more here

Reply:
Wait I am confused. We have one whole (already merged) issue about this change. From `tekton` to `oci`:. It will be reflected in the docs as follows: https://84253--ocpdocs-pr.netlify.app/openshift-pipelines/latest/secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.html#supported-parameters-tekton-chains-configuration_using-tekton-chains-for-openshift-pipelines-supply-chain-security

Reply:
The `artifacts.oci.storage` is already set to `oci` by default according to our docs, so no change there: https://docs.openshift.com/pipelines/1.16/secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.html#chains-supported-parameters-oci_using-tekton-chains-for-openshift-pipelines-supply-chain-security Am I missing something?